styles/base.css

```diff
@@ -3,10 +3,11 @@
 }
 
 body {
-  background-color: #332f3c;
+  background-color: #1f1c26;
   display: grid;
   font-family: "Poppins", sans-serif;
   font-size: 0.75rem;
+  color: #332f3c;
   grid-template-rows: 67px auto 724px;
```

Comment:
No, this are horrible grid rows. `724px` are you kidding me? ;-) Don't copy styles from Figma.